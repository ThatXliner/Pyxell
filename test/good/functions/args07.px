
func f([Float] x: [0]) def
    print x

f()
f(x=[8, 10^^16])
