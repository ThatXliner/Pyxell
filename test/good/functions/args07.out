[0]
[8, 1e+16]
